psst-gui/src/ui/search.rs

```diff
@@ -29,6 +29,9 @@ pub fn input_widget() -> impl Widget<AppState> {
     TextBox::new()
         .with_placeholder("Search")
         .controller(InputController::new().on_submit(|ctx, query, _| {
+            if query.trim().is_empty() {
+                return
```

Comment:
Minor nit, missing a `;` here?

Reply:
As lambda it works, but if you prefer I can add it :)